### ISSUE TITLE
Base64 support

### ISSUE THREAD

**Issue by terinjokes** [borkdude/jet#123]
`jq` has support for encoding and decoding base64 strings (`@base64` and `@base64d` respectfully). I often want to work with collections that have base64 strings, but found there's currently no support.

I've found that this does work (though very verbose) in Babashka, but not in jet.

```clojure
(String. (.. java.util.Base64 getDecoder (decode "TWFueSBoYW5kcyBtYWtlIGxpZ2h0IHdvcmsu")))
;"Many hands make light work."
```

**Comment by borkdude** [comment]
Is your proposal to make the babashka example work in jet, or to offer functions to decode and encode base64?

**Comment by terinjokes** [comment]
I would definitely prefer something shorter, if possible.

**Comment by borkdude** [comment]
We could add `base64/decode` and `base64/encode` (which go from string to string)?

**Comment by terinjokes** [comment]
That seems like a reasonable compromise for me.

**Comment by borkdude** [comment]
Available as jet 0.4.23
